core/src/com/unciv/logic/automation/ConstructionAutomation.kt

```diff
@@ -187,6 +186,11 @@ class ConstructionAutomation(val cityConstructions: CityConstructions){
         if (preferredVictoryType == VictoryType.Cultural
                 && wonder.name in listOf("Sistine Chapel", "Eiffel Tower", "Cristo Redentor", "Neuschwanstein", "Sydney Opera House"))
             return 3f
+        // Only start building if we are the city that would complete it the soonest
+        if (wonder.uniques.contains("Triggers a Cultural Victory upon completion") && cityInfo == civInfo.cities.minByOrNull {
+                it.cityConstructions.turnsToConstruction(it.cityConstructions.getCurrentConstruction().name) + it.cityConstructions.turnsToConstruction(wonder.name) 
```

Comment:
Again, I think we only need `it.cityConstructions.turnsToConstruction(wonder.name) ` and we can ignore the current construction.

Reply:
This calculation also includes the time it takes before we can start constructing the wonder. If our capital can build it in 10 turns, and we can build it in 15 turns, but our capital is still taking 8 turns building the manhatten project, we are faster to build it here than there: 15 turns instead of 18. If we were to replace the current building, this would of course be invalidated.
This is based on the assumption this function is only called whenever the old production is finished, is that assumption incorrect?

Reply:
This function is called every turn, for the AI, it can decide to switch productions in the middle
A building that basically done the game is definitely important enough to stop all other construction

Reply:
Alright, than you are completely correct, and I'll change it.